File: 158A - Next Round/main.php
<?php
 
/*
    https://codeforces.com/problemset/submission/158/225652762 (Web OUTPUT is 0? WTF, i don't know how to fix)
    PHP 8.1.7
*/
 
function input(){
    $f = fopen("php://stdin", "r");
    $str = fgets($f);
    fclose($f);
    return $str;
}
 
function nextround($n, $k) {
    if ($n < $k) {
        return 0;
    }
    $total = 0;
    $a = explode(" ", input());
    for($i = 0; $i < $n; $i++) {
        if (($a[$i] >= $a[$k - 1]) && ($a[$i] > 0)) {
            $total++;
        }
    }
    return $total;
}
 
$_str = explode(" ", input());
$n = $_str[0];
$k = $_str[1];
 
echo(nextround($n, $k));

?>

File: 158A - Next Round/main.py
# https://codeforces.com/problemset/submission/158/225645950
# Python 3.8.10 - 92 ms - 0 KB

def nextround() -> int:
    n, k = map(int, input().split())
    if n < k:
        return 0
    player = list(map(int, input().split()))
    total = 0
    for i in player:
        if i >= player[k - 1] and i > 0:
            total += 1        
    
    return total


if __name__ == '__main__':
    print(nextround())